src/main/java/org/opendevstack/provision/services/JenkinsPipelineAdapter.java

```diff
@@ -127,7 +127,7 @@
   protected String generalCdUser;
 
   @Value("${jenkinspipeline.create-project.default-project-groups:''}")
-  private String clusterAdminsEntitlementGroups;
+  protected String clusterAdminsEntitlementGroups;
```

Comment:
@stitakis Needed to be `protected` to set this value from the unit test

Reply:
@felipecruz91 actually `private` is the correct one. For the unit test we just need to add the property to the `application-utest.properties`. This is the on used by the `JenkinsPipelineAdapter` class. (see line 56 -> `@ActiveProfiles("utest")` ). I'll do this